python3-qtodotxt.json

```diff
@@ -0,0 +1,127 @@
+{
```

Comment:
The file has some misindentation, it would be nice if you could use `jq` (or anything else like this) to reformat it.

Reply:
Also manifest name has to match the application id.